karate-tests/src/test/java/mocks/sandbox/supporting-functions.js

```diff
@@ -52,6 +52,20 @@ function setInvalidSearchDataError (diagnostics) {
   response.status = 400
 }
 
+// eslint-disable-next-line no-unused-vars
+function setUnsupportedServiceError () {
+  response.body = context.read('classpath:mocks/stubs/errorResponses/UNSUPPORTED_SERVICE.json')
```

Comment:
I can't see a stub for UNSUPPORTED_SERVICE.json, apologies if I've missed it - can you confirm please?

Reply:
Also, for errorResponse, is missing X-requestID still tested for? Just double check to ensure all the previous error scenarios are covered sufficiently

Reply:
UNSUPPORTED_SERVICE.json does exist - but thanks for raising this, because while I was looking around I saw that the setUnsupportedServiceError function had been defined twice, so I fixed that ;-)

Reply:
The headers validation is checked by the validateHeaders function in supporting-functions.js. It's called by all of the endpoint handlers we have I believe, and there should be tests to cover all of this